ushahidi/platform-client
# Activity page

Master Task
https://github.com/ushahidi/platform/issues/436

Things to test-
1. Navigate to activity view
2. 'This Week' should be selected as the default period for the page
3. You should see data visualised in three visualisations/charts in this view - a line chart showing number of posts over time, a bar chart showing number of posts by categories, and a map showing posts
4. Check if the data shown is indeed for the selected time frame. The time frames that you can select are **This Week, This Month, All Time** (is data being shown consistent with the selected view?)
5. Check if charts labels make sense. Check for typos and illegibility when there is too much data being shown.
6. You should see upto 10 updates in the activity timeline aside component. Right now updates are only for new posts.

Notes:
~~1. There are no indicators for when data is loading or missing. You might just see blank charts for a few seconds while the data is loading. This is a TODO.~~
~~2. The recent activity timeline on right in non-functioning atm. This is a TODO~~
3. Charts showing most active posts and most active users is a pending item because it requires new API end points.
4. 'Today' time period is currently unavailable. Need to look into producing stats by the hour for this time range to be meaningful.


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/12)
